core/blockchain/stxo.go

```diff
@@ -99,6 +104,7 @@ func spentTxOutHeaderCode(stxo *SpentTxOut) uint64 {
 func spentTxOutSerializeSize(stxo *SpentTxOut) int {
 	size := serializeSizeVLQ(spentTxOutHeaderCode(stxo))
 	size += hash.HashSize
+	size += 8
```

Comment:
we need to avoid the hard coding the encode/decode the field

Reply:
https://github.com/Qitmeer/qitmeer/pull/259/commits/832ba31220c63893a51636c78bd49de85e07806e